### DISCUSSION TITLE
Change: Text ABOVE images, not below

### DISCUSSION THREAD

**Discussion by ReessKennedy** [glushchenko/fsnotes#1746] [Ideas]
Continued amazing work on this app. Here is something that has causes me some frustration with my idea for solution ... curious to know thoughts. 

## What
When I use the quick add in for IOS to add a photo or screenshot to a new note (which I often do) the text part is inserted below the image in the final note.

This causes a few issues I think could be solved just by altering this to insert the text ABOVE the image. Here is an image of what I mean and I will follow up with why I think this will solve some issues: 

<img width="929" alt="Screenshot 2024-07-16 at 12 24 48 PM" src="https://github.com/user-attachments/assets/ac5ca3e2-b54c-4423-bdf8-3b5bdd7d8507">

## Issue - File names
As seen in the photo, if you have FS Notes set to automatically rename files it does not choose the first bit of understandable text but sees the characters and resorts to numerical numbering when the first part of the note contains the special characters of a markdown image. If the text description were above, as I am proposing, this would not be the case. 

## Issue - Viewing image descriptions when scrolling and formatting issues
I sometimes witness formatting inconsistencies on image attachment previews -- either the images doesn't show or is blank. When I go into the note and manually change the position of the text to the top of note followed by the image, the formatting issue is resolved which leads me to believe it is also connected to beginning a note with "![]()" instead of some descriptive text.

**Comment by glushchenko** [comment]
Implemented, did you have testflight?

**Reply by ReessKennedy** [reply]
I do, yes. Happy to test out if you send invite or info on TestFlight build. 

So cool! I think this rearrangement will also help with an issue where images don't show up sometimes in the image preview until some text is manually move or entered above the image markdown link.

**Reply by glushchenko** [reply]
Please check TestFlight

**Reply by ReessKennedy** [reply]
Not sure how to add. When I open TestFlight I normally have to add an invitation code, right? Is there one for FS Notes or other method?

**Reply by glushchenko** [reply]
I've added your email
